CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Add `app.in_foreground` and `thread.main` flag to protocol. ([#1578](https://github.com/getsentry/relay/pull/1578))
 - Add support for View Hierarchy attachment_type. ([#1642](https://github.com/getsentry/relay/pull/1642))
 - Add invalid replay recording outcome. ([#1684]https://github.com/getsentry/relay/pull/1684)
+- Add max replay size configuration parameter. ([#1694]https://github.com/getsentry/relay/pull/1694)
```

Comment:
```suggestion
- Add max replay size configuration parameter. ([#1694](https://github.com/getsentry/relay/pull/1694))
```

And also one above this line should be changed to follow the proper format.

Reply:
Ah, I was wondering why the changelog ci was failing.  Thanks :D